packages/sdk-react/src/core/queries/useReceivables.ts

```diff
@@ -590,3 +590,31 @@ export const useReceivableContacts = (receivable_id: string) => {
 
   return useCounterpartContactList(receivable?.counterpart_id);
 };
+
+export const useMarkInvoiceAsPaid = (receivable_id: string) => {
+  const { i18n } = useLingui();
+  const { api, queryClient } = useMoniteContext();
+
+  return api.receivables.postReceivablesIdMarkAsPaid.useMutation(
+    {
+      path: {
+        receivable_id,
+      },
+    },
+    {
```

Comment:
It would be helpful to add error handling within the onError callback to notify users of any issues during the mutation

Reply:
I agree, good catch.